Gemfile

```diff
@@ -55,6 +55,9 @@ gem "record_tag_helper"
 gem "rinku", ">= 1.2.2", :require => "rails_rinku"
 gem "validates_email_format_of", ">= 1.5.1"
 
+#
+gem "quad_tile"
```

Comment:
This needs to be `~> 1.0.1` as the first version doesn't actually build.

Reply:
Addressed in 0a08659